test/steps/general/3_then.js

```diff
@@ -82,13 +97,21 @@ export function itShouldRejectWithTheErrorMessage() {
 export function itShouldRejectWithFileSystemErrorAndMessage() {
 	const { returnValue } = this.test.ctx;
 	const message = getFirstQuotedString(this.test.title);
-	return returnValue.should.be.rejectedWith(new FileSystemError(message));
+	return returnValue.should.be.rejected.then(err => {
+		err.should.be.instanceOf(Error);
+		err.should.have.property('name').which.equal('FileSystemError');
+		return err.should.have.property('message').which.include(message);
+	});
```

Comment:
Similar suggestion about adding a method to chai.

Reply:
great idea! it's much cleaner.
I will add `customError` check function